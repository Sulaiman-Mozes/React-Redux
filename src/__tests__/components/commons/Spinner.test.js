import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../../components/commons/Spinner';


describe('Spinner Component', () => {
  it('should render the Spinner Component', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
