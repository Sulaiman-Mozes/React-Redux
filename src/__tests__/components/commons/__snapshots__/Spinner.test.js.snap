// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should render the Spinner Component 1`] = `ShallowWrapper {}`;
